config/sync/user.role.site_editor.yml

```diff
@@ -0,0 +1,53 @@
+uuid: b35dd6b6-930d-4b4c-aeb2-443a0ba6fb59
+langcode: en
+status: true
+dependencies: {  }
+id: site_editor
+label: 'Site Editor'
+weight: -2
+is_admin: null
+permissions:
```

Comment:
- User can't unpublish any content.

Reply:
Thanks, I'll add the `Administer content` permission.